src/xunit.performance.analysis/Program.cs

```diff
@@ -318,6 +327,44 @@ private static void WriteTestResultsCSV(Dictionary<string, Dictionary<string, Te
             }
         }
 
+        private static void WriteStatisticsCSV(Dictionary<string, Dictionary<string, TestResult>> testResults, string analyzeOutputPath)
+        {
+            using (var writer = new StreamWriter(analyzeOutputPath))
+            {
+                writer.WriteLine("Test, Iterations, Duration Min, Duration Max, Duration Average, Duration Stdev, Metrics");
+                foreach (var run in testResults)
+                {
+                    foreach (var result in run.Value.Values)
+                    {
+                        RunningStatistics durationStats = result.Stats[DurationMetricName];
+                        writer.WriteLine(
+                            "\"{0}\",\"{1}\",\"{2}\",\"{3}\",\"{4}\",\"{5}\",\"{6}\"",
+                            EscapeCsvString(result.TestName),
+                            durationStats.Count,
+                            durationStats.Minimum,
+                            durationStats.Maximum,
+                            durationStats.Mean,
+                            durationStats.StandardDeviation,
+                            GetMetricsString(result.Stats.Keys)
```

Comment:
"GetMetricsString(result.Stats.Keys)"  if I understand correctly it should be the metrics which ideally should not have any special characters but just to be on the safer side...